README.md

```diff
@@ -394,6 +394,11 @@ server:
   # role to assume before querying EC2 API in order to discover metadata like EC2 private DNS Name
   ec2DescribeInstancesRoleARN: arn:aws:iam::000000000000:role/DescribeInstancesRole
 
+  # AWS Account IDs to scrub from server logs. (Defaults to empty list)
+  scrubbedAccounts:
```

Comment:
Nit: To me "redacted" is more explicit than "scrubbed"

Reply:
I went back and forth on this, but in my mind "redacted" sounded too much like the whole log entry was omitted, and in this case I'm just redacting part of the log line.

Reply:
I like and understand "scrub" FWIW. 👍